cli/lib/kontena/callbacks/exec_commands/option_placement.rb

```diff
@@ -0,0 +1,16 @@
+module Kontena
+  module Callbacks
+    class OptionPlacement < Kontena::Callback
+      matches_commands '* exec', '* ssh'
```

Comment:
The upside of this solution is that plugin commands such as `kontena vagrant master ssh` are now handled too.